BUILDING.md

```diff
@@ -1,43 +1,19 @@
 # Building OpenAssetIO
 
-## Quick start: Python only
-
-For use in a Python-only environment, the Python package can be
-installed from the root of the repository using
-
-```shell
-pip install .
-```
-
-This will automatically build and install the Python extension module,
-along with the Python sources.
-
-> **Note**
->
-> CMake's `find_package` must be able to locate dependencies.
-> Consider using the [`CMAKE_TOOLCHAIN_FILE`](https://cmake.org/cmake/help/v3.24/envvar/CMAKE_TOOLCHAIN_FILE.html)
-> environment variable to augment CMake's package search paths.
->
-> For example: to use `conan` to resolve dependencies, from the root of
-> the repository run (assuming a Linux host, translate as necessary):
->
-> ```shell
-> conan install -if .conan resources/build --build=missing
-> export CMAKE_TOOLCHAIN_FILE=`pwd`/.conan/conan_paths.cmake
-> pip install .
-> ```
-
-This will install a single binary module with the OpenAssetIO core
-library statically linked into it, and so is not suitable for use in a
-C++ application.
-
-For users that wish to modify the Python sources and test their changes
-without reinstalling, [editable installs](https://pip.pypa.io/en/stable/topics/local-project-installs/#editable-installs)
-are also supported
-
-```shell
-pip install --editable .
-```
+## Contents
+
+- [System requirements](#system-requirements)
```

Comment:
[question] Was this TOC generated, or did you create it manually? Would be good to know the auto-generator used so we can use it again when this doc is updated.  Not sure where to put that info though.

Reply:
It was autogenerated, but I don't think it's too important to record the generator, it was just from the top of google, they're a dime a dozen. (Good thing about being open source is not having to worry about using those sorts of things.)

I've massaged it myself by hand anyway after the fact when moving sections about.